miner/payload_building.go

```diff
@@ -262,6 +272,9 @@ func (miner *Miner) buildPayload(args *BuildPayloadArgs, witness bool) (*Payload
 					log.Info("Error while generating work", "id", payload.id, "err", r.err)
 				}
 				timer.Reset(miner.config.Recommit)
+			case inclusionList = <-payload.inclusionList:
+				fullParams.inclusionList = inclusionList
```

Comment:
I believe that this will cause Geth to execute full payload construction twice without being interrupted:  once when a new payload construction is requested via `fcu` and then again when the inclusion list is added.

Reply:
Isn't each payload construction interrupted by `miner.config.Recommit`?

Reply:
Ah right.